server/src/main/java/org/elasticsearch/cluster/metadata/DataLifecycle.java

```diff
@@ -33,10 +33,10 @@
  */
 public class DataLifecycle implements SimpleDiffable<DataLifecycle>, ToXContentObject {
 
-    public static final Setting<RolloverConditions> CLUSTER_DLM_DEFAULT_ROLLOVER_SETTING = new Setting<>(
+    public static final Setting<RolloverConfiguration> CLUSTER_DLM_DEFAULT_ROLLOVER_SETTING = new Setting<>(
         "cluster.dlm.default.rollover",
         "max_age=7d,max_primary_shard_size=50gb,min_docs=1,max_primary_shard_docs=200000000",
```

Comment:
we should update the default value to have `max_age=auto` as well

Reply:
Well spotted, done.